### DISCUSSION TITLE
About the API in transplanting project to mitsuba3

### DISCUSSION THREAD

**Discussion by Whisper-MMM** [mitsuba-renderer/mitsuba3#963] [Q&A] [answered]
Thanks for the great work in mitsuba3

I'm working on a transplant of some projects in mitsuba2 to 3. However, some errors occur with the attribute "AnimatedTransform." I checked the doc in mitsuba2 and found it encapsulates an animated 4x4 homogeneous coordinate transformation. So may I know some alternative functions in mitsuba3?

**Comment by merlinND** [comment, marked as answer]
Hello @Whisper-MMM,

In Mitsuba 2, the `AnimatedTransform` type was exposed but never really used (e.g. motion blur support was not implemented).
I believe that in Mitsuba 3, we removed the bindings for that type until it receives proper support.
You should be able to replace them with `mi.Transform4f` (assuming your project was not actually using their features).
